### ISSUE TITLE
Ignore Untrusted SSL Flag

### ISSUE THREAD

**Issue by billabongrob** [panodata/grafana-wtf#31]
It would be helpful to ignore untrusted SSL certificates.  Merely a quality of life enhancement.

**Comment by amotl** [comment]
Dear Rob,

there is a hack to set the `CURL_CA_BUNDLE` environment variable to an empty string, as outlined at [^1]. It can be verified with
```shell
$ CURL_CA_BUNDLE="" python3 -c "import requests; requests.get('https://quad9.net')"
```
```python
/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py:1013:
  InsecureRequestWarning: Unverified HTTPS request is being made to host 'quad9.net'.
  Adding certificate verification is strongly advised.
  See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
```

That would work out-of-the-box until the package has grown a non-hacky solution.

With kind regards,
Andreas.

[^1]: https://stackoverflow.com/questions/48391750/disable-python-requests-ssl-validation-for-an-imported-module/48391751#48391751

**Comment by amotl** [comment]
Dear Rob,

#35 finally implements your suggestion, it has been released with version 0.13.3. Thank you again!

It can be used by appending the `?verify=no` query string to the `GRAFANA_URL`, like:

    export GRAFANA_URL=https://daq.example.org/grafana/?verify=no

With kind regards,
Andreas.
